test/pkg/report/features.go

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License.AGPL.txt in the project root for license information.
+
+package report
+
+type Feature string
+
+const (
+	FeatureNone          Feature = "none"
```

Comment:
Probably we don't require this one. We should try to find a feature sets that the integration tests belong to.

Reply:
Make sense. Deleted it